package.json

```diff
@@ -28,18 +28,19 @@
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
     "request": "^2.87.0",
+    "swagger-ui-express": "^4.0.6",
```

Comment:
Move `swagger-ui-express` to devDependencies

Reply:
okay